README.md

```diff
@@ -1,5 +1,5 @@
 # Instructions
 
 1. Clone this repository
-2. Checkout to a branch which has your name. Example: Deyan Dobrinov => `deyan-dobrinov`
-3. Solve questions [1; N] available in the root directory
+2. Create a new branch, use your first and last name. Example: Deyan Dobrinov => `deyan-dobrinov`
```

Comment:
My previous explanation was probably not good enough. I am usually creating a branch for the candidate who I am going to interview in advance, so the branch exists. In this branch I am usually committing few symbolic links to question which I would like to include in the interview. Basically the idea here is to see the interviewee perform some basic operations with Git. Nothing else.